src/containerapp/azext_containerapp/_help.py

```diff
@@ -1915,6 +1915,139 @@
           az containerapp connected-env list -g MyResourceGroup
 """
 
+helps['containerapp connected-env dapr-component'] = """
+    type: group
+    short-summary: Commands to manage Dapr components for Container Apps connected environments.
+"""
+
+helps['containerapp connected-env dapr-component list'] = """
+    type: command
+    short-summary: List Dapr components for a connected environment.
+    examples:
+    - name: List Dapr components for a connected environment.
+      text: |
+          az containerapp connected-env dapr-component list -g MyResourceGroup --name MyConnectedEnv
+"""
+
+helps['containerapp connected-env dapr-component show'] = """
+    type: command
+    short-summary: Show the details of a Dapr component.
+    examples:
+    - name: Show the details of a Dapr component.
+      text: |
+          az containerapp connected-env dapr-component show -g MyResourceGroup --dapr-component-name MyDaprComponentName --name MyConnectedEnv
+"""
+
+helps['containerapp connected-env dapr-component set'] = """
+    type: command
+    short-summary: Create or update a Dapr component.
+    examples:
+    - name: Create a Dapr component.
+      text: |
+          az containerapp connected-env dapr-component set -g MyResourceGroup --name MyEnv --yaml MyYAMLPath --dapr-component-name MyDaprComponentName
+"""
+
+helps['containerapp connected-env dapr-component remove'] = """
```

Comment:
should we name with delete？

Reply:
It is consisting with `containerapp env dapr-component remove`